### PR TITLE
NAS-102000

### PR DESCRIPTION
**Pull request by dkmullen**
Also completes NAS-102623
Replaces form array with form list in Network interfaces, show failover/virtual failover addresses when needed, add style, add slash and default netmask of 24, etc.

### CODE REVIEW

**Comment by lola27** [comment]
@dkmullen could you please resolve the conflicts? thanks

File: src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts [hunk 1]
```diff
@@ -261,6 +263,23 @@ export class InterfacesFormComponent implements OnDestroy {
     this.vlan_pint = _.find(this.fieldConfig, {'name' : 'vlan_parent_interface'});
     this.bridge_members = _.find(this.fieldConfig, {'name' : 'bridge_members'});
     this.lag_ports = _.find(this.fieldConfig, {'name' : 'lag_ports'});
+
+    if (window.localStorage.getItem('is_freenas') === 'false') {
+      this.ws.call('failover.node').subscribe((node) => {
+        if (node === 'A') {
+          this.ipPlaceholder = ' (This Controller)';
```
Comment (wblock) [inline src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts]: Shouldn't this use the global text here?

Comment (dkmullen) [reply src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts]: I updated 'This Controller'  to use global variable.

File: src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts [hunk 2]
```diff
@@ -337,22 +263,41 @@ export class InterfacesFormComponent implements OnDestroy {
...
+    if (window.localStorage.getItem('is_freenas') === 'false') {
+      this.ws.call('failover.node').subscribe((node) => {
+        if (node === 'A') {
+          this.ipPlaceholder = ` (${globalHelptext.thisCtlr})`;
+          this.failoverPlaceholder = ` (${globalHelptext.Ctrlr} 2)`;
+        } else if (node === 'B') {
+          this.ipPlaceholder = ` (${globalHelptext.Ctrlr} 1)`;
+          this.failoverPlaceholder = ` (${globalHelptext.thisCtlr})`;
+        } else {
+          this.ipPlaceholder = ' The active controller cannot be detected.';
```
Comment (wblock) [inline src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts]: Does this need a T() for translation?  Should it be in the global helptext file?

File: src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts [hunk 3]
```diff
@@ -168,118 +165,75 @@ export class InterfacesFormComponent implements OnDestroy {
...
-        name: 'delete',
-        placeholder: helptext.delete_placeholder,
-        tooltip: helptext.delete_tooltip,
-        isHidden: true,
-        disabled: true,
-      }]
-    },
+      type: 'list',
+      name: 'aliases',
+      placeholder: 'Aliases',
```
[the remark this answers is not in the document]
Comment (dkmullen) [reply src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts]: done

File: src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts (hunk 2 above)
Comment (dkmullen) [reply src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts]: done

**Pull request merged**